src/integration/java/io/pinecone/helpers/TestIndexResourcesManager.java

```diff
@@ -37,13 +39,17 @@ public class TestIndexResourcesManager {
     private Pinecone pineconeClient;
     private String podIndexName;
     private IndexModel podIndexModel;
-    private String serverlessIndexName;
+    private IndexModel serverlessIndexModel;
+    private  String serverlessIndexName;
     private String collectionName;
     private CollectionModel collectionModel;
-    private final List<String> podIndexVectorIds = Arrays.asList("id1", "id2", "id3");
+    private final List<String> vectorIds = Arrays.asList("id1", "id2", "cidddd3");
```

Comment:
Changed so can be index-type agnostic

Reply:
Just so it's really clear what the purpose of the prefix is, maybe just use `prefix-id3`?